worklets/Overview.bs

```diff
@@ -410,7 +412,33 @@ loops and callbacks exceeding imposed time limits.
 Security Considerations {#security-considerations}
 ==================================================
 
-Issue(w3c/css-houdini-drafts#92): Need to decide if to allow worklets for unsecure context, etc.
+Worklets do not introduce any usable API surface to the web themselves, instead other
+specifications, e.g. [[css-paint-api-1]], [[webaudio]] will extend this specification to provide
+useful functionality.
+
+Worklets are allowed to be used outside <a>SecureContexts</a> as specifications which extend/use
+worklets may be allowed to used outside <a>SecureContexts</a>.
```

Comment:
Right - I don't think we have consensus as user agents yet that this is the case for all new features, which is why I worded it this way.

I've changed it to 
"Specifications which use worklets should decide if their worklet(s) should be allowed outside SecureContexts". instead?